packages/experimental/Text/src/Text.tsx

```diff
@@ -1,67 +1,25 @@
 /** @jsx withSlots */
 import {
-  FontTokens,
-  FontVariantTokens,
   fontStyles,
   withSlots,
-  buildUseTokens,
   UseTokens,
   useFluentTheme,
-  applyTokenLayers,
   mergeStyles,
   compressible,
   patchTokens,
 } from '@fluentui-react-native/framework';
-import { Text as RNText, ColorValue } from 'react-native';
-import { ITextProps } from '@fluentui-react-native/adapters';
+import { Text as RNText } from 'react-native';
+import { textName, TextProps, useTextTokens, TextTokens } from './Text.types';
 import React from 'react';
 
-const textName = 'Text';
-
-/**
- * Text tokens, these are the internally configurable values for Text elements. In particular these
- * drive decisions on how to build the styles
- */
-export interface TextTokens extends FontTokens {
-  /** foreground text color */
-  color?: ColorValue;
-
-  /** alternate mode for disabled look and feel */
-  disabled?: TextTokens;
-}
-
-/**
- * Text props, based off of the standard react-native TextProps with some new extensions
- */
-export type TextProps<TBase = ITextProps> = TBase &
-  FontVariantTokens & {
-    /** foreground text color */
-    color?: ColorValue;
-
-    /** whether or not this text should be presented as disabled */
-    disabled?: boolean;
-  };
-
-const useTextTokens = buildUseTokens<TextTokens>(
```

Comment:
How is this building without useTextTokens?

Also. it seems like useTextTokens is supplying some defaults and props can be used to override them, so I think it would make sense to keep it and use it to fill defaults from tokens.

Reply:
I actually moved it to the newly created Text.types file. Do you think it makes more sense to be placed here?

Reply:
A .types file should only contain types or interface declarations. I don't think it should hold any functional code.
If you'd like the styling to be in its own file, we have Component.styling.ts ComponentTokens.ts in other components that I think would fit this better.